### PR TITLE
Added aria-label to MultivaluedStringComponent 

### PR DESCRIPTION
**Pull request by raphisuter**
## Motivation
Adding aria-label to MultivaluedStringComponent to get rid of error in DevTools Console. Was moved out of PR https://github.com/keycloak/keycloak-ui/pull/4306 to separate PR.

## Checklist:
- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

File: apps/admin-ui/src/components/dynamic/MultivaluedStringComponent.tsx [hunk 1]
```diff
@@ -26,11 +26,13 @@ export const MultiValuedStringComponent = ({
     >
       <MultiLineInput
         name={fieldName}
+        aria-label={t(label!)}
         isDisabled={isDisabled}
         defaultValue={[defaultValue]}
         addButtonLabel={t("addMultivaluedLabel", {
           fieldLabel: t(label!).toLowerCase(),
         })}
+        stringify
```
Comment (jonkoops) [inline apps/admin-ui/src/components/dynamic/MultivaluedStringComponent.tsx:35]: Can we get rid of this unrelated change?

**Comment by raphisuter** [comment]
closed for new PR

File: apps/admin-ui/src/components/dynamic/MultivaluedStringComponent.tsx (hunk 1 above)
Comment (raphisuter) [reply apps/admin-ui/src/components/dynamic/MultivaluedStringComponent.tsx:35]: my bad, instead of doing reverts here I created a new branch without other commits on it to start from a clean base. closed this PR.
